packages/ctl/ctl.js

```diff
@@ -43,7 +43,11 @@ function formatOutputColored(output) {
 	let info = "";
 	if (output.type === "log") {
 		let level = output.level;
-		if (level === "Info") {
+		if (level === "Script") {
```

Comment:
The logging levels are better done as constants. You know how it goes, you think it's gonna be used only once and suddenly strings are all over the code :)